src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -89,6 +89,7 @@ let nextId = 0;
   templateUrl: './flyout.component.html',
   styleUrls: ['./flyout.component.scss'],
   providers: [
+    SkyFlyoutAdapterService,
```

Comment:
Now that the adapter is provided by the component, we no longer need to generate a renderer using the `Renderer2Factory`.
Instead of this:
```
private renderer: Renderer2;

constructor(
  private rendererFactory: RendererFactory2
) {
  this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
}
```

We can now do this:
```
constructor(
  private renderer: Renderer2
) {}
```